furnace/google_scholar_paper.py

```
@@ -1,10 +1,29 @@
+import random
```

Ruff (0.8.2)

`random` imported but unused

Remove unused import: `random`

(F401)